deploy/role.yaml

```diff
@@ -52,3 +52,438 @@ rules:
   - '*'
   verbs:
   - '*'
+
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: knative-eventing-operator
+
+rules:
+
+  #######################
+  ### EVENTING - CORE
```

Comment:
@jcrossley3 I've added all the different `apiGroups` here from the different `ClusterRole` kinds from Eventing and Eventing-Sources.

However, one file I did NOT include...:

* since those have some labels/annotations, see:  https://github.com/knative/eventing/blob/v0.5.0/config/200-addressable-resolvers-clusterrole.yaml#L22-L33

Reply:
yeah, olm doesn't support aggregation rules. we just have to do it manually based on expected use